src/app/(private)/roles/[id]/page.tsx

```diff
@@ -348,7 +358,7 @@ const Page: NextPage<{ params: { slug: string; id: string } }> = ({
             description: t('role.activatedSuccessfully'),
             variant: 'success',
           })
-          router.push('/roles')
+          setRefreshData((prev) => !prev)
         } else {
```

Comment:
burada neden tersini alıyoruz? success ise refresh gerekiyor diyip "true" diyemez miyiz? veya success ise direkt tekrar fetchlesek state tutmadan?

Reply:
useEffect'i tetiklemesi için ekledim. !prev kısmı sadece tetiklensin diye kullandım. Sadece true da denebilirdi ama state'in değişmesini istediğim için tersini almak daha mantıklı geldi.